src/cli/cli_mac_filter.cpp

```diff
@@ -218,6 +260,23 @@ template <> otError MacFilter::Process<Cmd("rss")>(Arg aArgs[])
             error = otLinkFilterAddRssIn(GetInstancePtr(), &extAddr, rss);
         }
     }
+    /**
+     * @cli macfilter rss add
+     * @code
+     * macfilter rss add * -50
+     * Done
+     * @endcode
+     * @code
+     * macfilter rss add 0f6127e33af6b404 -85
+     * Done
+     * @endcode
+     * @cparam macfilter rss add @ca{extaddr} @ca{rss}
+     * @par api_copy
+     * #otLinkFilterAddRssIn
+     * @par
+     * To set a default value for the received signal strength for all received messages,
```

Comment:
It might be better to have this sentence (To set a default value...) after the cparam line, since it's specific to the parameter usage.  So it would be just text on its own after cparam.  We've done this before, like with: https://github.com/openthread/openthread/blob/main/src/cli/cli.cpp#L4005.

Thoughts?